federation/cmd/federation-controller-manager/app/options/options.go

```diff
@@ -35,6 +35,10 @@ type ControllerManagerConfiguration struct {
 	Port int `json:"port"`
 	// address is the IP address to serve on (set to 0.0.0.0 for all interfaces).
 	Address string `json:"address"`
+	// federation name.
+	FederationName string `json:"federationName"`
+	// zone name, like example.com.
+	ZoneName string `json:"zoneName"`
```

Comment:
Which zone is this? The zone in which federation control plane is running?
k8s clusters could be in multiple zones.